test/functional/test_framework/util.py

```diff
@@ -496,6 +496,58 @@ def check_node_connections(*, node, num_in, num_out):
     assert_equal(info["connections_in"], num_in)
     assert_equal(info["connections_out"], num_out)
 
+# Assumes -maxmempool=5(or similarly small).
+# It will ensure mempools become synced as it is based on a single node
+# and assumes -minrelaytxfee is 1sat/vB.
+def fill_mempool(test_framework, node, miniwallet):
+        """Fill mempool until eviction.
+
+        Allows for simpler testing of scenarios with minfee != minrelay
+        Requires -datacarriersize=100000 and
+        -maxmempool=5(or similarly small).
+        It will not ensure mempools become synced as it
+        is based on a single node and assumes -minrelaytxfee
+        is 1 sat/vbyte.
+        """
+        test_framework.log.info("Fill the mempool until eviction is triggered and the mempoolminfee rises")
+        txouts = gen_return_txouts()
+        relayfee = node.getnetworkinfo()['relayfee']
+
+        tx_batch_size = 1
+        num_of_batches = 75
+        # Generate UTXOs to flood the mempool
+        # 1 to create a tx initially that will be evicted from the mempool later
+        # 75 transactions each with a fee rate higher than the previous one
+        # And 1 more to verify that this tx does not get added to the mempool with a fee rate less than the mempoolminfee
+        # And 2 more for the package cpfp test
+        test_framework.generate(miniwallet, 1 + (num_of_batches * tx_batch_size))
+
+        # Mine 99 blocks so that the UTXOs are allowed to be spent
+        test_framework.generate(node, 99)
```

Comment:
In "Move fill_mempool to util function" (cc770e4b36e6f04cbd588ad3e7a19f122c21d38d):

Nit: If a commit is described as a move, I would expect no changes in the code beyond what’s necessary to update the callsites. While these changes are minor and obviously do not entail a functional change, I would have expected the updated commentary and refactor to be a separate commit—I’m wondering whether my understanding of the modus operandi is correct.

Reply:
split out the moving and docstring changes, should be minimal now?

Reply:
Nit: I don’t feel strongly about it, but the change that made me notice that there were code changes in the move originally was that  `COINBASE_MATURITY - 1` had gotten replaced with a `99` magic number.

Reply:
bit of an annoying circular reference issue(blocktools requiring utils) so for now at least make it grep-able what the magic value should be(and more greppable)